sources/iTermMetalFrameData.m

```diff
@@ -76,7 +76,7 @@ void iTermMetalFrameDataStatsBundleInitialize(iTermPreciseTimerStats *bundle) {
         "DrawCursor<<<<",
         "DrawMarks<<<<",
         "DrawCrGuide<<<<",
-        "DrawHighlite<<<<",
+        "DrawHighlight<<<<",
```

Comment:
This was abbreviated because of a space limitation

Reply:
Interesting, do note that `"EndEncodingInt<<<<` is longer...

Reply:
That one's OK to rename, then. Sorry for the error!